src/scss/02_atoms/_buttons.scss

```diff
@@ -266,22 +269,25 @@ $downloadButtonConfig: (
   padding-left: get($buttonSizeConfig, $size, paddingLeft);
   padding-right: get($buttonSizeConfig, $size, paddingRight);
   padding-top: get($buttonSizeConfig, $size, paddingTop);
+
+  .btn__text {
+    line-height: get($buttonSizeConfig, $size, lineHeight);
+  }
 }
 
 .btn {
   @include buttonSize(base);
-  align-content: center;
-  align-items: center;
-  align-self: center;
   border-style: solid;
   cursor: pointer;
-  display: flex;
+  display: inline-block;
```

Comment:
Io personalmente manterrei il flex come prima, se proprio serve mettiamo in `inline-flex`. L'inline-block da sempre alcuni margini automatici che eviterei.